Moya/Moya.swift

```diff
@@ -175,6 +177,9 @@ private extension MoyaProvider {
         // We need to keep a reference to the closure without a reference to ourself.
         let networkActivityCallback = networkActivityClosure
 
+        //Print network request 
```

Comment:
Let's not use the word "print", since a logger could do _anything_. A better comment could be:

```swift
// Log network request.
```

(Note the space after the `//` and punctuation – it's _super_ not a big deal, but I'd like to keep it consistent. Goes for the other additions to the file, too.)